packages/button/test/button.test.ts

```diff
@@ -423,6 +423,45 @@ describe('Button', () => {
             expect(el.getAttribute('aria-label')).to.equal('clickable');
         });
 
+        it('updates aria-label when label changes', async () => {
+            const el = await fixture<Button>(html`
+                <sp-button label="Initial Label">Button</sp-button>
```

Comment:
```suggestion
                <sp-button label="Initial label">Button</sp-button>
```

(Super minor nit - we try to keep with content standards for Spectrum which includes Sentence case and not title case. It's a label though so it's really not that important.)